mitodl/micromasters
# Increase socket-timeout

#### What are the relevant tickets?
fixes #1971 

#### What's this PR do?
Increases `socket-timeout` uwsgi setting to help prevent timeout

#### How should this be manually tested?
In Chrome turn on GPRS throttling and upload a 2mb profile image. It should take a while but it should complete without error.
